pingdom-checks/patient-generated-data.ping

```diff
@@ -10,3 +10,12 @@ pingdom save-check \
   -a authorization_token="$HEALTH_APIS_STATIC_ACCESS_TOKEN" \
   -a userids_csv="$NO_USERS" \
   -a integrationids_csv="$PGD_SLACK_ID"
+
+pingdom save-check \
+  --template fhir-resource \
+  -a name=production-pgd-r4-questionnaire \
+  -a host=api.va.gov \
+  -a url="/services/pgd/v0/r4/Questionnaire/37953b72-961b-41ee-bd05-86c62bacc46b" \
+  -a authorization_token="$HEALTH_APIS_STATIC_ACCESS_TOKEN" \
+  -a userids_csv="$NO_USERS" \
```

Comment:
^ will this eventually be hooked up to Statuspage?

Reply:
Most likely, I'll check with Gabe